### ISSUE TITLE
questions about batchnorm in DistributedDataParallel

### ISSUE THREAD

**Issue by ZGCTroy** [MCG-NJU/TDN#22]
hi, thanks for sharing the code! 
You have batchnorm in your resnet model, and also use DistributedDataParallel. I want to know if it is necessary to use SyncBatchnorm to have a more accurate result. I'm always confused about this kind of details.

To replace Batchnorm with SyncBatchnorm in DDP, only two lines of code need to change: 
" model = torch.nn.SyncBatchNorm.convert_sync_batchnorm(model)" and "isinstance(m, torch.nn.BatchNorm2d) or SyncBatchnorm:"  in get_optim_policies. However, applying SyncBatchnorm may lead to a bit of decrease in speed.

**Comment by yztongzhan** [comment]
Hi @ZGCTroy, thanks for your suggestion.
For fair comparison with other 2D-based methods, we don't use the `SyncBatchnorm` for training. 
However you can try it by yourself for better performance :)
